### PR TITLE
Add default launch screen storyboard to test app hosts 

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Add default launch screen to test app hosts  
  [dnkoutso](https://github.com/dnkoutso)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: spec/unit/generator/info_plist_file_spec.rb [hunk 1]
```diff
@@ -31,42 +31,61 @@ module Pod
...
-          'CFBundleInfoDictionaryVersion' => '6.0',
-          'CFBundleName' => '${PRODUCT_NAME}',
-          'CFBundlePackageType' => 'APPL',
-          'CFBundleShortVersionString' => '1.0.0',
-          'CFBundleSignature' => '????',
-          'CFBundleVersion' => '${CURRENT_PROJECT_VERSION}',
-          'NSPrincipalClass' => 'NSApplication',
-        }
-      end
+    it 'includes additional entries if requested' do
```
Comment (dnkoutso) [inline spec/unit/generator/info_plist_file_spec.rb]: new test

File: lib/cocoapods/generator/app_target_helper.rb [hunk 2]
```diff
@@ -90,10 +90,34 @@ def self.add_empty_swift_file(project, target, name = 'App')
...
+      # @param  [Symbol] platform
+      #         the platform of the target. Can be `:ios` or `:osx`, etc.
+      #
+      # @param  [String] name
+      #         The name to use for the target, defaults to 'App'.
+      #
+      # @return [PBXFileReference] the created file reference of the launchscreen storyboard.
+      #
+      def self.add_launchscreen_storyboard(project, target, name = 'App')
+        launch_storyboard_file = AppTargetHelper.create_launchscreen_storyboard_file(project, name)
```
Comment (segiddins) [inline lib/cocoapods/generator/app_target_helper.rb]: This doesn't need to reference `AppTargetHelper` directly

Comment (dnkoutso) [reply lib/cocoapods/generator/app_target_helper.rb]: ah the other ones do, I copied pasted. Meh

**Pull request merged**